tempodb/tempodb.go

```diff
@@ -389,101 +375,18 @@ func (rw *readerWriter) maintenanceLoop() {
 }
 
 func (rw *readerWriter) pollBlocklist() {
-	start := time.Now()
-	defer func() { metricBlocklistPollDuration.Observe(time.Since(start).Seconds()) }()
+	blocklist, compactedBlocklist, err := rw.blocklistPoller.Do()
```

Comment:
super minor nit: `readerWriter` is doing the polling so its more of a block lister than a block poller.. :P